src/common/environment-helper.ts

```diff
@@ -0,0 +1,17 @@
+// production environment= "production"
+// local development = "development"
+// preview release = "staging"
+// feature branch (github PRs)= "feature"
+// test environment = "test"
+
+import { IS_TEST_ENV } from './constants';
+
+function getEnvironment() {
+  if (IS_TEST_ENV) return 'test';
+  if (process.env.NODE_ENV === 'development') return 'development';
+  if (!process.env.PREVIEW_RELEASE && !process.env.IS_PUBLISHING) return 'feature';
+  if (process.env.PREVIEW_RELEASE) return 'staging';
+  return 'production';
+}
+
+export const ENVIRONMENT = getEnvironment();
```

Comment:
Could we avoid using `IS_TEST_ENV` (use the env var directly), so we can export this from the constants file (avoiding a circular dep)?

Reply:
Good idea !